### PR TITLE
Create ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57339](https://cumulocity.atlassian.net/browse/MTM-57339)
Original PR: [5926](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5926)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Ng cli prompt verison limit is now 5. Additionally, the latest tag is now shown for the correct version.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: ng-cli prompt version limit increased to 5 and latest tag now shown for correct version
```

File: content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57339
+version: 1019.13.3
+---
+Ng cli prompt verison limit is now 5. Additionally, the latest tag is now shown for the correct version.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md:17]: The Claude AI suggested to use the following body:
```suggestion
The ng-cli prompt, which is used to create and manage Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Angular versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Angular version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of Angular, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the ng-cli prompt.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by janhommes** [comment]
@Enio94 can you review my suggestion if this is correct?

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57339
+version: 1019.13.3
+---
+The Angular CLI new command, which is used to create and manage Web SDK based Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Web SDK versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Web SDK version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of the Web SDK, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the Angular CLI new comand prompt.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md:17]:
```suggestion
The Angular CLI new command, which is used to create and manage Web SDK-based Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Web SDK versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Web SDK version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of the Web SDK, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the Angular CLI new command prompt.
```

File: content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The version selection is more clear when scaffolding a new application with Angular CLI
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md:3]:
```suggestion
title: Clearer version selection when scaffolding a new application with Angular CLI
```

**Pull request merged**